src/steps/02-EvaluationCriteria/MRR/ReviewBarriers.vue

```diff
@@ -7,19 +7,187 @@
             Let’s review your agency’s plans to remove barriers to fair opportunity
           </h1>
           <p>
-            Intro paragraph
+            Based on what you’ve told us, we’ve suggested language to describe the actions proposed
+            to remove barriers to fair opportunity. You can edit any details to meet your
+            requirements.
           </p>
+          <ATATTextArea
+            class="textarea-max-width"
+            id="BarriersTextArea"
+            :value.sync="barriersToOpportunity"
+            :maxChars="2500"
+            :rows="getRowCount"
+            :validateItOnBlur="true"
+            :rules="[
+                $validators.required('Describe the actions proposed to remove barriers to fair' +
+                 ' opportunity..'),
+                $validators.maxLength(2500)
```

Comment:
Need a more specific validation message for max length. This is not captured in the story but captured in FIGMA. I caught this based on a similar feedback I also received :-)
"
$validators.maxLength(
                  2500,
                  'Limit your description to 2,500 characters or less.'
                ), 
"